src/react/components/pages/projectSettings/projectSettingsPage.tsx

```diff
@@ -67,14 +67,11 @@ export default class ProjectSettingsPage extends React.Component<IProjectSetting
         const projectToUpdate: IProject = {
             ...formData,
         };
-
-        const isNew = !(!!projectToUpdate.id);
-        await this.props.actions.saveProject(projectToUpdate);
-
-        if (isNew) {
+        try {
+            await this.props.actions.saveProject(projectToUpdate);
             this.props.history.push(`/projects/${this.props.project.id}/edit`);
-        } else {
-            this.props.history.goBack();
+        } catch (e) {
+            console.log(e);
```

Comment:
Remove console.log for now and let error bubble... We'll handle the error handling in a future story.